sdk/src/Services/DynamoDBv2/Custom/AWSConfigs.DynamoDB.cs

```diff
@@ -167,6 +167,11 @@ public partial class DynamoDBContextConfig
         /// </remarks>
         public bool? DisableFetchingTableMetadata { get; set; }
 
+        /// <summary>
+        /// If true, all <see cref="DateTime"/> properties are retrieved in UTC timezone while reading data from DynamoDB. Else, the local timezone is used.
```

Comment:
Can you make this clear this is for the high level library. The service client itself is going to always use UTC.